### DISCUSSION TITLE
Error during loading of plugin

### DISCUSSION THREAD

**Discussion by D0zee** [eclipse/tm4e#575] [Q&A]
Hello everyone! I've made plugin. It works perfect locally, but when I load it on update site, it gives me strange error which is connected with **tm4e**. My plugin is very simple - it's just a some provider that launch some jar. [This is the repo of project ](https://github.com/D0zee/Eclipse-Gradle-KTS-editing/tree/master), master branch. 

The plugin contains of two subprojects: [feature](https://github.com/D0zee/Eclipse-Gradle-KTS-editing/tree/master/org.eclipse.buildship.gradleprop.feature) which just include one plugin, and [plugin itself](https://github.com/D0zee/Eclipse-Gradle-KTS-editing/tree/master/org.eclipse.buildship.gradleprop.provider). The stack trace of error I've attached as screen. 

The most strange is that I don't use **org.eclipse.e4.ui.workbench**. Maybe you can find the mistake in my project, thank you for all answers in advance!

<img width="850" alt="Снимок экрана 2023-08-02 в 14 08 17" src="https://github.com/eclipse/tm4e/assets/90183817/51d4bdea-4784-4d96-b3d8-c4c50c02795d">

**Comment by sebthom** [comment]
I guess you are for some reason using mismatching versions of the tm4e.core and tm4e.ui plugins.

These combinations work for me:
![image](https://github.com/eclipse/tm4e/assets/426959/b4ae80f6-8689-4d65-a076-11e55b27fd2a)

![image](https://github.com/eclipse/tm4e/assets/426959/f3073962-6325-460a-b0f3-cd9702494ae3)
